taskvine/src/manager/vine_current_transfers.c

```diff
@@ -84,6 +84,22 @@ int vine_current_transfers_dest_in_use(struct vine_manager *q, struct vine_worke
 	return c;
 }
 
+// return true if worker w is currently a receiving file cachename, else false
+int vine_current_file_worker_receiving(struct vine_manager *q, struct vine_worker_info *w, const char *cachename)
```

Comment:
Must be called `vine_current_transfers_` something because it is a public member of `vine_current_transfers.h`